lib/list.js

```diff
@@ -75,4 +78,84 @@ function buildUrl (opts) {
   return fullURL;
 }
 
+const COLLECTIONS_MAPPING = ['ds:3', 0, 1];
+const CLUSTER_URL_MAPPING = [0, 0, 3, 4, 2];
+const CLUSTER_NAME_MAPPING = [0, 1];
+const CLUSTER_NAMES = {
+  TOP_FREE: 'top free apps',
```

Comment:
Gotcha, you make the first request in English to get the cluster names, right?

Reply:
That's right, I checked and the clusters are the dale regardless of language.